src/Level06/Conf/CommandLine.hs

```diff
@@ -36,7 +36,7 @@ partialConfParser =
 
 -- Parse the Port value off the command line args and into a Last wrapper.
 portParser
```

Comment:
Haddock's not quite right, and the `Last` in the middle feels like a detail that the `portParser` shouldn't care about. Maybe `fmap` the `Last` in when you assemble the whole parser?